pixi.toml

```diff
@@ -1,20 +1,38 @@
 [project]
-name = "gluex-nstar"
-version = "0.1.0"
-description = "Add a short description here"
 authors = ["Vitor Shen <17490173+shenvitor@users.noreply.github.com>"]
 channels = ["conda-forge"]
+name = "gluex-nstar"
 platforms = ["linux-64"]
 
 [tasks]
+docnb = """
+sphinx-build \
+    -nW --keep-going \
+    -b html \
+    docs _build/html
```

Comment:
This is okay, but for consistency with other ComPWA projects, I would write the files to `docs/_build/html`. Be careful to also update that output path in the upload path of the `build` job.